CubeLineMoment.py

```diff
@@ -214,9 +201,9 @@ def cubelinemoment_setup(cube, cuberegion, cutoutcube,
     brightest_cube = cutoutVcube.spectral_slab(vz-velocity_half_range,
                                                vz+velocity_half_range)
 
-    # compute various moments & statistics along the spcetral dimension
+    # compute various moments & statistics along the spectral dimension
     peak_velocity = brightest_cube.spectral_axis[brightest_cube.argmax(axis=0)]
-    max_map = peak_amplitude = brightest_cube.max(axis=0)
+    max_map = peak_amplitude = brightest_cube.max(axis=0) # This sometimes contains an all-NaN slice
```

Comment:
Maybe add "(and that's OK)" if it is, which I think we've agreed it is?

Reply:
Agreed.